crates/bermuda/src/lib.rs

```diff
@@ -58,35 +68,97 @@ fn triangulate_path_edge(
         limit.unwrap_or(3.0),
         bevel.unwrap_or(false),
     );
-    let triangle_data: Vec<u32> = result
+    triangulation_to_edge_triangulate(py, &result)
+}
+
+fn triangulation_to_edge_triangulate(
```

Comment:
Would it be more accurate to name `triangulation_to_edge_triangulation` since this seems to convert the data into a suitable python format?

Reply:
Perhaps add a 1 line function docstring `Convert triangulation data to a Python edge triangulation`

Reply:
changed name